validation-test/Concurrency/issue70019.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -O -Xfrontend -disable-availability-checking %s -parse-as-library -module-name main -o %t/main
+// RUN: %target-codesign %t/main
+// RUN: %target-run %t/main | %FileCheck %s
+
+// REQUIRES: concurrency
+// REQUIRES: executable_test
+// REQUIRES: concurrency_runtime
+// REQUIRES: optimized_stdlib
+
+// https://github.com/apple/swift/issues/70019
+
+@usableFromInline
+struct Foo {
+    var integer: Int
+}
+
+struct Bar: Sendable {
+    var foo: Foo // expected-warning: Stored property 'foo' of 'Sendable'-conforming struct 'Bar' has non-sendable type 'Foo'
```

Comment:
The message you are checking for has to be wrapped in {{}} and ‘:’ is extraneous.